cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -544,4 +544,48 @@ describe('Dashboards', () => {
       })
     })
   })
+
+  before('Clear downloads folder', () => {
+    cy.exec('rm cypress/downloads/*', {log: true, failOnNonZeroExit: false})
+  })
+  it('creates a dashboard and downloads JSON', () => {
+    cy.get('@org').then(({id: orgID}: Organization) => {
+      cy.createDashboard(orgID).then(({body}) => {
+        cy.fixture('routes').then(({orgs}) => {
+          cy.visit(`${orgs}/${orgID}/dashboards/${body.id}`)
+          cy.getByTestID('tree-nav')
+          cy.getByTestID('nav-item-dashboards').click()
+          cy.getByTestID('dashboard-card').invoke('hover')
+          cy.getByTestID('context-export-menu').click()
+          cy.getByTestID('context-menu-item-export').click()
+          cy.getByTestID('button').click()
+          cy.wait(1000)
```

Comment:
lets try and find another way to wait here other than a fixed time amount. Could we use `cy.intercept` to wait on the download request to finish?

Reply:
I decided to delete cy.wait from the code, since every command has a 4s timeout by Cypress' default configuration.
If the downloaded file is not read by cy.readfile in this time range, the test will fail.
Also, I was unable to use cy.intercept, because the file is not stored on a server. The download is provided by a blob.